### ISSUE TITLE
PayTab version 2

### ISSUE THREAD

**Issue by Parth-Patoliya** [paytabscom/paytabs-ios-library-sample#43]
Is pod 'PayTabs', '~> 4.3.1' 
Connected to Pay tab v2 portal :  https://merchant.paytabs.com/merchant/login

As still these SDK ask for Secret key and Region.
While there is no Secret key in new portal.

Also the documentation link here is not redirecting to proper place.

**Comment by MuhamedAdly** [comment]
Yes, it is connected to PT2, you can use your old credentials of PT1 or get it by navigating to the "INTEGRATION MAPPING" under the "Developers" section.
